guides/doc-Planning_Guide/topics/Glossary.adoc

```diff
@@ -156,7 +158,9 @@ To see the full list of services, enter the `{foreman-maintain} service list` co
 
 [[varl-Glossary_of_Terms-Foreman_Hooks]]
 *Foreman Hook*:: An executable that is automatically triggered when an orchestration event occurs, such as when a host is created or when provisioning of a host has completed.
-
++
+Note that Foreman Hook functionality is deprecated and will be removed in the next {Project} version.
```

Comment:
I think this is true for downstream only. While I think @lzap considers the classic hooks deprecated (in favor of webhooks), we don't ship it by default in upstream. That means it's not really removed as such. Also, you're still able to install it in upstream.

Reply:
If it's for downstream, wrap like this: 
```suggestion
ifdef::satellite[]
Note that Foreman Hook functionality is deprecated and will be removed in the next {Project} version.
endif::[]
```

Reply:
Changed as per comment.